k8s/test/e2e/assembly/distributed-assembly-e2e.yaml

```diff
@@ -25,23 +25,19 @@ setup:
         - namespace: cert-manager
           resource: pod
           for: condition=Ready
-    - name: install scheduler-plugin and operator
+    - name: install vineyard operator
       command: |
-        kind load docker-image vineyardcloudnative/vineyard-operator:latest
+        docker tag vineyardcloudnative/vineyard-operator:latest localhost:5001/vineyard-operator:latest
+        docker push localhost:5001/vineyard-operator:latest
+        export IMG=localhost:5001/vineyard-operator:latest
```

Comment:
Why not `make install-vineyard-operator` as you have already added the target in Makefile?